### PR TITLE
Remove FRI custom module

### PR DESCRIPTION
**Pull request by esoergel**
Pending confirmation from Jeremy
@gcapalbo @nickpell

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1680,9 +1674,7 @@ def _determine_couch_databases(couch_databases):
 }
 
 # These are custom templates which can wrap default the sms/chat.html template
-CUSTOM_CHAT_TEMPLATES = {
-    "FRI": "fri/chat.html",
-}
```
Comment (esoergel) [inline settings.py]: @gcapalbo it didn't look like this was overrideable in ansible, so I think this is only used by FRI.  There's also a lot of code associated with using and updating this, should that also be all removed?

File: custom/fri/_design/views/extra_message/map.js [hunk 2]
```diff
@@ -1,5 +0,0 @@
-function(doc) {
-    if(doc.doc_type === "FRIExtraMessage") {
-        emit([doc.domain, doc.message_id], null);
-    }
-}
```
Comment (esoergel) [inline custom/fri/_design/views/extra_message/map.js]: @dannyroberts do you recall what else should happen when we delete couch views?  Just wait for compaction to take care of it, or is some manual step or migration necessary?

**Comment by orangejenny** [comment]
hooray, I'm highly in favor of this "delete old custom code" initiative

File: custom/fri/models.py [hunk 3]
```diff
@@ -1,60 +0,0 @@
...
-    PROFILE_C: "C - IDU",
-    PROFILE_D: "D - PSE/CSV",
-    PROFILE_E: "E - Top",
-    PROFILE_F: "F - Bottom",
-    PROFILE_G: "G - General",
-    PROFILE_H: "H - Other",
-}
-
-
-class FRIMessageBankMessage(Document):
```
Comment (emord) [inline custom/fri/models.py]: Should there be a management command to delete these old docs when we delete these?

File: custom/fri/_design/views/extra_message/map.js (hunk 2 above)
Comment (dannyroberts) [reply custom/fri/_design/views/extra_message/map.js]: Great memory! That might have slipped my notice.

Yes. After deploying (and being sure we're not reverting!) you can run

```
cchq <env> django-mange prune_couch_views
```

File: custom/fri/models.py (hunk 3 above)
Comment (esoergel) [reply custom/fri/models.py]: Hmm, yeah, probably - although then we wouldn't ever be able to revert.  I'll ask Jeremy what he thinks about that.  Might have to provide a dump or something.

Comment (gcapalbo) [reply custom/fri/models.py]: i started an email thread about this. depending on the answer to that, all the fri_* fields on the SMSBase class might also go

File: settings.py (hunk 1 above)
Comment (gcapalbo) [reply settings.py]: i think for now we'd want to just remove the entry in this dictionary, but not the setting itself. but yes, in general, if product agrees we might be able to remove the custom chat template feature and some other sms chat-related features that were built only for this project

**Review by gcapalbo — APPROVED** [review]

**Pull request merged**